dirsrvtests/tests/suites/acl/misc_test.py

```diff
@@ -499,6 +504,54 @@ def finofaci():
         Accounts(conn, DN).filter("(objectclass=top)", scope=ldap.SCOPE_ONELEVEL, strict=True)
 
 
+def test_delete_aci_with_invalid_syntax(topo, request):
+    """Verify that an ACI with invalid syntax can be deleted after being imported
+
+    :id: 7f3e8a9b-4c2d-4e5f-9a1b-2c3d4e5f6a7b
+    :setup: Standalone Instance
+    :steps:
+        1. Import an LDIF containing an ACI with invalid syntax
+        2. Attempt to delete the invalid ACI
+    :expectedresults:
+        1. Import should succeed
+        2. Delete operation should succeed without error
+    """
+    inst = topo.standalone
+
+    invalid_aci = ('(targetattr ="fffff")(version 3.0;acl "Directory Administrators Group";'
+                   'allow (all) (groupdn = "ldap:///cn=Directory Administrators, dc=example,dc=com");)')
+    valid_aci = ('(targetattr!="userPassword")(version 3.0; acl "Enable anonymous access"; '
+                 'allow (read, search, compare) userdn="ldap:///anyone";)')
+
+    log.info("Create LDIF file with invalid ACI")
+    ldif_file = f'{inst.get_ldif_dir()}/invalid_aci_test.ldif'
+    with open(ldif_file, 'w') as f:
+        f.write(f"dn: {DEFAULT_SUFFIX}\n")
+        f.write("objectClass: top\n")
+        f.write("objectClass: domain\n")
+        f.write("dc: example\n")
+        f.write(f"aci: {valid_aci}\n")
+        f.write(f"aci: {invalid_aci}\n")
+    os.chmod(ldif_file, 0o777)
+
+    def fin():
+        if os.path.exists(ldif_file):
+            os.remove(ldif_file)
+    request.addfinalizer(fin)
+
+    log.info("Import LDIF with invalid ACI")
+    import_task = ImportTask(inst)
+    import_task.import_suffix_from_ldif(ldiffile=ldif_file, suffix=DEFAULT_SUFFIX)
+    import_task.wait()
```

Comment:
I think we need to check the import task exit status here - `import_task.get_exit_code()`